From: Ross Gammon <rossgammon@mail.dk>
Date: Sat, 25 Jan 2014 16:48:20 +0100
Subject: Disable HTML View

* gramps/plugins/view/htmlrenderer.py: Comment out import of webkit module

Origin: upstream, suggestion in INSTALL file if webkit unstable
---
 gramps/plugins/view/htmlrenderer.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/gramps/plugins/view/htmlrenderer.py b/gramps/plugins/view/htmlrenderer.py
index c41942a..8f35b16 100644
--- a/gramps/plugins/view/htmlrenderer.py
+++ b/gramps/plugins/view/htmlrenderer.py
@@ -122,11 +122,12 @@ user_pref("general.useragent.locale, %(lang)s);
 
 TOOLKIT = NOWEB
 
-try:
-    from gi.repository import WebKit as webkit
-    TOOLKIT = WEBKIT
-except:
-    pass
+#Disable webkit as it is causing a crash in Gramps 4.0.2
+#try:
+#    from gi.repository import WebKit as webkit
+#    TOOLKIT = WEBKIT
+#except:
+#    pass
 
 #no interfaces present, raise Error so that options for GeoView do not show
 if TOOLKIT == NOWEB :
